pkg/controllers/cidrs_controller.go

```diff
@@ -104,24 +105,73 @@ func (r *CIDRReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 }
 
 func applyProcessor(reader io.Reader, processing ipamv1alpha1.Processing) ([]string, error) {
-	if processing.JSONPath != "" {
+	// Default to YAML format if not specified (backward compatibility)
+	format := processing.Format
+	if format == "" {
+		format = ipamv1alpha1.YAML
+	}
+
+	switch format {
+	// Handle simple text formats CSV/LVS
+	case ipamv1alpha1.CommaSeparatedValues:
+		return processCSV(reader, processing)
+	// Handle YAML format (existing logic)
+	case ipamv1alpha1.YAML:
+		return processYAMLFormat(reader, processing)
+	default:
+		return nil, fmt.Errorf("unsupported format: %s", format)
+	}
+}
+
+func processCSV(reader io.Reader, processing ipamv1alpha1.Processing) ([]string, error) {
+	data := csv.NewReader(reader)
+	data.Comment = '#'        // Ignore comments
+	data.FieldsPerRecord = -1 // Disable strict field count checking
+	data.LazyQuotes = true    // Allow unescaped quotes inside fields
+	data.TrimLeadingSpace = true
+	csvArray, err := data.ReadAll()
+	if err != nil {
+		return nil, fmt.Errorf("failed to read comma-separated values: %w", err)
+	}
+
+	var cidrs []string
+	for _, records := range csvArray {
+		for _, record := range records {
+			if record != "" {
+				cidr := strings.TrimSpace(record)
+				_, _, err := net.ParseCIDR(cidr)
```

Comment:
err is ignored here, we should report it at least as a warning (log is not enough as the owner of the CIDR may not have access to the controller logs)

Reply:
hmm, should we report about err ?
CSV with descriptions will always generate error for description part.